memprof.c

```diff
@@ -850,8 +850,9 @@ static char * generate_filename(const char * format) {
 
 static void memprof_zend_error_cb_dump(MEMPROF_ZEND_ERROR_CB_ARGS)
 {
-	char * filename = NULL;
-	php_stream * stream;
+	const char * output_format = MEMPROF_G(output_format);
+	char * filename = generate_filename(output_format);
+	php_stream * stream = php_stream_open_wrapper_ex(filename, "w", 0, NULL, NULL);
```

Comment:
These may do memory allocations and need to be in the `WITHOUT_MALLOC_TRACKING` block